### ISSUE TITLE
[popup] "Unexpected token Delim('.')" when packaging Fomantic-UI css distribution with Parcel.js

### ISSUE THREAD

**Issue by mhthies** [fomantic/Fomantic-UI#2315]
# Bug Report
Minifying/packaging Fomantic-UI using the latest version of [Parcel.js](https://parceljs.org/) and the `fomantic-ui-css` package from NPM fails with the following error:

```
@parcel/transformer-css: Unexpected token Delim('.')

  /home/michael/Documents/Freizeit/Programme/python/shc2/node_modules/fomantic-ui-css/semantic.css:61341:36
    61340 | 
  > 61341 | [data-tooltip][data-inverted]:after .header {
  >       |                                    ^
    61342 |   background: none;
    61343 |   color: #FFFFFF;
```

## Steps to reproduce
1. Install the latest Parcel.js and fomantic-ui-css packages from NPM:
   `package.json`:
   ```json
   {
     "dependencies": {
       "parcel": "^2.4.1",
       "fomantic-ui-css": "^2.8.8"
     }
   }
   ```
   ```bash
   npm install
   ```
2. Create a simple `main.js` file as an entry point for Parcel.js:
   ```js
   import 'fomantic-ui-css/semantic.css';
   import 'fomantic-ui-css/semantic.js';
   ```
3. Package project with Parcel.js:
   ```bash
   npx parcel build main.js
   ```

## Expected result
Packaging should complete successful.

## Actual result
Parcel.js fails with the error message shown above.

## Version
2.8.8

## Further Thoughts

The issue seems to be the following CSS/LESS code from [`popup.less`, line 246](https://github.com/fomantic/Fomantic-UI/blob/develop/src/definitions/modules/popup.less#L246):
```css
[data-tooltip][data-inverted]::after .header {
  background: @tooltipInvertedHeaderBackground;
  color: @tooltipInvertedHeaderColor;
}
```
I'm not sure, if this is actually invalid CSS. But it doesn't look sensible either: AFAIK, the `::after` pseudo-elements cannot contain nested elements, so I understand that Parcel.js complains about the `...:after .header` structure.

**Comment by lubber-de** [comment]
Should be fixed by #2316
Try to remove the whole selector as the PR did, if that makes parceljs satisfied.

**Comment by mhthies** [comment]
Thank you!

I already fixed it temporarily by deleting the selector; however this is somehow inconvenient to automate in an automated build. So for now, I'll stick with the older Parcel.js version until the next Fomantic-UI release or the Parcel.js update is absolutely necessary.
